pkg/services/action_test.go

```diff
@@ -30,7 +30,7 @@ func (s ServicesTestSuite) TestNewTakeAction() {
 		failFetch := func(ctx context.Context, id uuid.UUID) (*models.SystemIntake, error) {
 			return nil, errors.New("error")
 		}
-		createAction := NewTakeAction(failFetch, submit, review, review, review, review, review, review)
+		createAction := NewTakeAction(failFetch, submit, review, review, review, review, review, review, review)
```

Comment:
I know this is... not great, in terms of scalability. It's on my to-do list, don't worry

Reply:
It's getting the job done for now. :-)